Regression in resolution change detection

In the build supporting 1.1.0 it seems that https://github.com/yamashi/CyberEngineTweaks/issues/193 is no longer fixed

I did add the required Release() calls but it still shows the same issue, hence I wanted to debug the code. The problem I now have is that the xmake f -c command failed to pull TiltedCore ` => download https://github.com/tiltedphoques/TiltedCore/releases/download/v0.1.5/release.zip .. failed` and every subsequent xmake -f c simply skips that. xmake f -m debug + xmake -y dies in stdafx.h now
```
C:\Users\frans.SD\Documents\GitHub\CyberEngineTweaks>xmake -y
[  0%]: compiling.release src\stdafx.h
error: stdafx.h
C:\Users\frans.SD\AppData\Local\.xmake\packages\t\tiltedcore\v0.1.5\8574311f4b544fa89a3d8c3ee79d1929\include\TiltedCore\Stl.hpp(5): fatal error C1083: Cannot open include file: 'tsl/hopscotch_map.h': No such file or directory
```
I'll retry from scratch

--- a/src/d3d12/D3D12.h
+++ b/src/d3d12/D3D12.h
@@ -32,9 +32,8 @@ protected:
 
     struct FrameContext
     {
-        ID3D12CommandAllocator* CommandAllocator { nullptr };
-        ID3D12Resource* BackBuffer { nullptr };
-        ID3D12Resource* MainRenderTargetResource { nullptr };
+        CComPtr<ID3D12CommandAllocator> CommandAllocator { nullptr };
+        CComPtr<ID3D12Resource> BackBuffer { nullptr };
         D3D12_CPU_DESCRIPTOR_HANDLE MainRenderTargetDescriptor{ 0 };
     };
 
@@ -64,10 +63,10 @@ private:
     std::vector<FrameContext> m_frameContexts{ };
     std::vector<ID3D12Resource*> m_downlevelBackbuffers{ };
     IDXGISwapChain3* m_pdxgiSwapChain{ nullptr };
-    ID3D12Device* m_pd3d12Device{ nullptr };
-    ID3D12DescriptorHeap* m_pd3dRtvDescHeap{ nullptr };
-    ID3D12DescriptorHeap* m_pd3dSrvDescHeap{ nullptr };
-    ID3D12GraphicsCommandList* m_pd3dCommandList{ nullptr };
+    CComPtr<ID3D12Device> m_pd3d12Device{ nullptr };
+    CComPtr<ID3D12DescriptorHeap> m_pd3dRtvDescHeap{ nullptr };
+    CComPtr<ID3D12DescriptorHeap> m_pd3dSrvDescHeap{ nullptr };
+    CComPtr<ID3D12GraphicsCommandList> m_pd3dCommandList{ nullptr };
     ID3D12CommandQueue* m_pCommandQueue{ nullptr };
     uint32_t m_downlevelBufferIndex{ 0 };
     

--- a/src/d3d12/D3D12_Functions.cpp
+++ b/src/d3d12/D3D12_Functions.cpp
@@ -6,7 +6,6 @@
 #include <imgui_impl/dx12.h>
 #include <imgui_impl/win32.h>
 
-#include <scripting/LuaVM.h>
 #include <window/Window.h>
 #include "Options.h"
 

--- a/src/dllmain.cpp
+++ b/src/dllmain.cpp
@@ -5,6 +5,10 @@
 #include "Image.h"
 #include "Options.h"
 
+#ifndef NDEBUG
+#include "scripting/GameHooks.h"
+#endif
+
 void EnableDebugPatch(const Image* apImage);
 void StartScreenPatch(const Image* apImage);
 void RemovePedsPatch(const Image* apImage);

--- a/src/scripting/Scripting.cpp
+++ b/src/scripting/Scripting.cpp
@@ -20,6 +20,7 @@
 #include "Utils.h"
 
 #ifndef NDEBUG
+#include "GameHooks.h"
 #include "GameDump.h"
 #include <RED4ext/Dump/Reflection.hpp>
 #endif

